src/hotspot/share/logging/logOutput.cpp

```diff
@@ -65,16 +65,16 @@ void LogOutput::add_to_config_string(const LogSelection& selection) {
 }
 
 
-static int tag_cmp(const void *a, const void *b) {
-  return static_cast<const LogTagType*>(a) - static_cast<const LogTagType*>(b);
+static int tag_cmp(const LogTagType *a, const LogTagType *b) {
+  return primitive_compare(a, b);
 }
 
 static void sort_tags(LogTagType tags[LogTag::MaxTags]) {
   size_t ntags = 0;
   while (tags[ntags] != LogTag::__NO_TAG) {
     ntags++;
   }
-  qsort(tags, ntags, sizeof(*tags), tag_cmp);
+  qsort(tags, ntags, sizeof(*tags), (_sort_Fn)tag_cmp);
```

Comment:
While wondering about, then searching for, _sort_Fn, I found it in globalDefinitions.hpp and a wrong comment in instanceKlass.cpp
```
1781:    // _sort_Fn is defined in growableArray.hpp.
```
would you mind fixing that comment or removing it?

Reply:
I removed it, since it seems not helpful.  Thanks for reviewing!